docs/process-definition/incubation.md

```diff
@@ -0,0 +1,71 @@
+[branches]: branches.md
+[branches stable]: branches.md#how-to-determine-it-is-stable
+[incubation releases]: releases.md#release-of-incubation
+[triage]: triage.md
+[tracker]: https://github.com/opendatahub-io/odh-dashboard/issues?q=is%3Aopen+is%3Aissue+label%3Atracker
+[templates]: https://github.com/opendatahub-io/odh-dashboard/issues/new/choose
+
+# Incubation
+
+Incubation is a term we use for flows that are not quite fully stable. They have been worked on by the developers and have yielded a flow that may not have all the edge cases figured out, or simply may just not be fully vetted for the flow. Essentially, we are looking for feedback on how the functionality works and if you find any use-cases you might be experiencing that does not quite work with the incubated features.
+
+## Branching Strategy
+
+There is a lot to discuss here, more detailed information can be found in the [branches] readme, but simply put, the flow follows this pattern:
+
+![branchesBasic.png](meta%2FbranchesBasic.png)
+
+- Features are created off the state of `main`
+- Code is added to the feature branches through PRs
+- The feature branches merge into `incubation` when they are ready
+- The feature branches get updates through testing / incubation efforts and merge these new changes into `incubation`
```

Comment:
I know we've discussed this but I'm still convinced that SMEs/future leads should be updating feature branches during incubation time periodically (could be once a sprint or smth). The reason behind is to detect conflicts in main. This conflicts are gonna be present when we merge main into incubation too since incubation is a superset of the other feature branches, but if we just fix incubation, the conflict will be there when merging the feature branch back to main, that's why I think that should be a good practice to follow.

Reply:
It won't stop conflicts, but it could be a good idea to do that. I'm not sure how we'd enforce that... perhaps it's a task for leads and we setup a slack reminder or something. I'll not mention it in this pass of the docs, we can discuss internally and figure out the best way to go about that.

Reply:
Yeah, I think this is a good practice, but I don't think we can get away from having to solve the conflicts multiple times. This should at least reduce the likelihood of being held up by multiple conflicts at critical periods, so still a really good idea.